tardis/visualization/tools/interaction_radius_plot.py

```diff
@@ -0,0 +1,461 @@
+import tardis.visualization.tools.sdec_plot as sdec
+
+import numpy as np
+import pandas as pd
+import astropy.units as u
+
+from tardis.util.base import (
+    atomic_number2element_symbol,
+    element_symbol2atomic_number,
+    species_string_to_tuple,
+    roman_to_int,
+    int_to_roman,
+)
+
+import matplotlib.pyplot as plt
+import matplotlib.cm as cm
+import matplotlib.colors as clr
+import plotly.graph_objects as go
```

Comment:
[nitpick] a good practice is to group imports in the follow following pattern:

imports from other packages

imports from local package (tardis)